stan/math/prim/scal/prob/poisson_log_cdf_log.hpp

```diff
@@ -0,0 +1,83 @@
+#ifndef STAN_MATH_PRIM_SCAL_PROB_POISSON_LOG_CDF_LOG_HPP
```

Comment:
Move this logic to poisson_log_lcdf.hpp. Change the logic of this file to call poisson_log_lcdf.